README.md

```diff
@@ -2,7 +2,30 @@
 
 This gem is used to report opinionated statistics about modularization to DataDog and other observability systems.
 
+# Setup and package.yml file standard
```

Comment:
Thank you for adding more information about setting up code ownership 🙏 

To avoid repeating any information in https://github.com/rubyatscale/code_ownership, can we link to https://github.com/rubyatscale/code_ownership#package-based-ownership, nix "common problems" below, and make this section a bit shorter (I like the idea of keeping the context that modularization statistics "reports metrics per-team")?

What do ya think?